polaris-react/src/components/CheckableButton/CheckableButton.scss

```diff
@@ -40,13 +40,15 @@
 
 .Checkbox {
   pointer-events: none;
```

Comment:
This bug was ocurring both with the feature flag disabled and enabled. 
This is a more holistic solution to the problem that also cleans up our css.